src/diem/offchain/types/command_types.py

```diff
@@ -134,3 +160,18 @@ class CommandResponseObject:
     error: typing.Optional[OffChainErrorObject] = datafield(default=None)
     # The Command identifier to which this is a response.
     cid: typing.Optional[str] = datafield(default=None)
+    # An result JSON object that may be defined when status == "success"
+    result: typing.Optional[OffChainResultObject] = datafield(default=None)
```

Comment:
use `dict  # pyre-ignore` for type definition.
when decoding the response object, result is unknown, as caller should base on request command type to decide what is response type, thus it is unknown when we decode CommandResponseObject.
For example: PaymentCommand response should not have result = ReferenceIDCommandResponse.
It is more flexible to let caller to decode the dict to specific type.